### DISCUSSION TITLE
Siguientes pasos (marzo 2021)

### DISCUSSION THREAD

**Discussion by nukeador** [PucelaBits/websegura#149] [General]
Abro hilo para hablar sobre los siguientes pasos con este proyecto, creo que ahora mismo hay tres cosas que creo son importantes:

-  Deberíamos aumentar la repercusión y presión pública a los organismos, necesitamos mejorar y aumentar las interacciones en twitter. ¿Qué más podríamos hacer dentro y fuera de la web?
- Es importante mostrar el impacto. ¿Están mejorando las notas a raíz de nuestro trabajo?
- Aún quedan provincias que no hemos cubierto, ¿cómo completamos el listado de provincias?
- Ampliar y colaborar, ¿cómo podemos involucrar a grupos locales?

¿Qué pensáis?

**Comment by palmerabollo** [comment]
De acuerdo en los 2 primeros puntos:
- La temática de la web es de nicho, no es super sexy, pero se puede probar la parte de la gamificación. Creo que en este tipo de sitios es complicado mantener el interés a lo largo del tiempo.
- Se puede dar un empujón haciendo la famosa lista de partidos políticos, u otra que se nos ocurra más interesante que organismos de los que muchas veces no habíamos ni oído hablar. Por probar.
- También creo que es importante el segundo punto (mostrar el impacto) porque 1. internet está lleno de _haters_ y gente que se queja de todo y a veces "se gana más con una cucharada de miel que con un barril de hiel" y 2. el objetivo del proyecto no es ser un escaparate de lo mal que están las cosas sino una forma de conseguir que mejoren.
- Lo de las provincias, quedan unas 20, nos las podemos repartir. Deberíamos poder encontrar 5-10 voluntarios en el grupo de pucelabits.
- Lo de los grupos locales: ni idea :) ¿Conocéis asociaciones parecidas en otras provincias?
